ibis/backends/flink/__init__.py

```diff
@@ -145,6 +145,34 @@ def list_tables(
         # but executing the SQL string directly yields a `TableResult` object
         return self._filter_with_like(tables, like)
 
+    def _list_views(
+        self,
+        like: str | None = None,
+        temporary: bool = True,
+    ) -> list[str]:
+        """Return the list of view names.
+
+        Return the list of view names in the specified database and catalog.
+        or the default one if no database/catalog is specified.
```

Comment:
Non-blocker since this method is now private, but should it accept `database` and `catalog` kwargs?

Reply:
turns out pyflink API doesn't expose database & catalog filters, (the underlying JNI API does but it may not be necessary to expose that for now). I'll update the docstring.